utils/vlog/vlog_to_pbtype.py

```diff
@@ -51,7 +51,9 @@
 
 import yosys.run
 from yosys.json import YosysJSON
-import xmlinc
+
+sys.path.insert(0, "..")
```

Comment:
There is not a reason to have sys.path modifications anymore.  Change the PYTHONPATH prior to invocation in CMake or install to site packages e.g. fasm & ice40.